cdap-ui/app/cdap/components/Reports/ReportsDetail/RunsPagination/index.js

```diff
@@ -35,7 +38,12 @@ function RunsPaginationView({totalCount, offset, limit}) {
       handlePageChange={handleRunsPageChange}
       currentPage={currentPage}
       totalPages={totalPages}
-      title={totalCount > 1 ? "Runs" : "Run"}
+      title={
+        totalCount > 1 ?
+          T.translate(`${PREFIX}.runPlural`)
+        :
+          T.translate(`${PREFIX}.runSingular`)
```

Comment:
use context for pluralization.

in i18n:
```
runs:
    "0..1": "{context} run"
    _: "{context} runs"
```

Can you update the rest of these that have plurals?

Reply:
Ok have used context for pluralization. 0 is plural though 😂